tables/he-common-consonants.uti

```diff
@@ -0,0 +1,90 @@
+
+# liblouis: hebrew-common-characters
+#
+#-display-name: Hebrew Common Characters
+#-index-name: he-common-characters
+#+language: he
+#+type:literary
+#+contraction:no
+#+system:IHBC
+#+dots:6
+
+#-license: lgpl-2.1
+
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+#This is a common character file that supports Hebrew forward translation. Matthew Yeater, Paul Geoghegan, and Leonard De Ruijter. 
+
+# always op code characters
+always אּ 5-1  # alef with dagesh
+always בְּ 12-3  # bet with dagesh sheva
+always בּ 12  # bet with dagesh
+always גּ 5-1245  # gimel with dagesh
+always דּ 5-145  # dalet with dagesh
+always הּ 5-125  # hey with dagesh
+always זּ 5-1356  # zayin with dagesh
+always חּ 5-1346  # het with dagesh
+always טּ 5-2345  # tet with dagesh
+always י 245  # Yod with dagesh
+always יּ 5-245  # Yod with dagesh
+always כּ 13  # kaf with dagesh
+always לּ 5-123  # lamed with dagesh
+always מּ 5-134  # mem with dagesh
+always נּ 5-1345  # nun with dagesh
+always סּ 5-234  # samekh
+always עּ 5-1246  # ayin with dagesh
+always פּ 1234  # pe with dagesh
+always צּ 5-2346  # tsadi with dagesh
+always קּ 5-12345  # qof with dagesh
+always רּ 5-1235  # Resh with dagesh
+always שּׁ 5-146  # shin with dot and dagesh
+always שּׂ 156  # sin with dagesh
```

Comment:
This is the wrong line with shin and dagesh

Reply:
FIxed